ml-agents-envs/setup.py

```diff
@@ -51,6 +51,7 @@ def run(self):
         "numpy>=1.14.1,<2.0",
         "Pillow>=4.2.1",
         "protobuf>=3.6",
+        "pyyaml",
```

Comment:
Not too happy about this...

Reply:
why not specify a lower bound for pyyaml version ?

Reply:
I copied this from the mlagents setup.py file

Reply:
copying from mlagents setup.py doesn't make it correct. I suggest having a lower bound on the pyyaml version in both places.